.github/workflows/interop.yml

```diff
@@ -17,6 +17,12 @@ jobs:
     name: Run transport interoperability tests
     runs-on: ubuntu-22.04
     steps:
+      - name: Free Disk Space
+        # For some reason the original job (libp2p/test-plans) has enough disk space, but this one doesn't.
```

Comment:
The disk space isn't related to the action on libp2p/test-plans. This comment is misleading, please remove it.

Reply:
We can say this is a workaround for a lack of smaller images for GitHub runners https://github.com/actions/runner-images/issues/10414.

Reply:
I find hard to believe that we're the only repo having issues with HDD space if all of us have access to the same kinds of runners, hence my quite generic comment.

I've updated it to be even more generic, removing the reference to `libp2p/test-plans`. If you feel like it doesn't fit I'd appreciate if you could provide one that does and I'll update it.